templates/blog/astro/minimal/src/styles/global.css

```
@@ -1 +1,14 @@
 /* === ブログコンテンツのスタイリングなどがあればここに === */
+/* === Opt-in ユーティリティ（lism-css コアには含まれない。https://lism-css.com/docs/utility-class/ の Opt-in 参照） === */
+@layer lism-utility {
+  /* 子要素の間に区切り線を表示する */
+  .u--divide {
+    --bdw: 1px;
+    --bdc: var(--divider);
+    gap: var(--bdw);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)